components/breadcrumbs/breadcrumbs.js

```diff
@@ -24,9 +25,8 @@ class Breadcrumbs extends LocalizeCoreElement(RtlMixin(LitElement)) {
 				display: block;
 				font-size: 0.7rem;
 				line-height: 1.05rem;
-				overflow: hidden;
 				position: relative;
-				white-space: nowrap;
+				${overflowEllipsisDeclarations}
```

Comment:
I know it would severely increase the complexity of this change, but wondering if we should be flagging it. We could use a similar approach [to the styling tweaks](https://github.com/BrightspaceUI/core/pull/5617/files).